node/network/protocol/src/peer_set.rs

```diff
@@ -56,10 +67,15 @@ impl PeerSet {
 				notifications_protocol: protocol,
 				max_notification_size,
 				set_config: SetConfig {
-					in_peers: 25,
+					// Collators don't need to accept incoming connections on this peer set:
```

Comment:
```suggestion
					// Non-authority nodes don't need to accept incoming connections on this peer set:
```

Full nodes are also non authority nodes :D

Reply:
And at some point we may also support collators who are also relay chain validators.